src/test/java/com/xceptance/neodymium/util/SelenideAddonsTest.java

```diff
@@ -36,99 +36,100 @@
+        $("#searchForm input").should(SelenideAddons.matchAttribute("placeholder", "\\D+"));
     }
 
     @Test
     public void testMatchAttributeConditionError()
     {
-        openBlogPage();
-        $("#masthead .search-toggle").click();
+        openPostersStartPage();
+        $("#header-search-trigger").click();
 
         Assert.assertThrows(ElementShould.class, () -> {
-            $("#search-container .search-field").should(SelenideAddons.matchAttribute("placeholder", "\\d+"));
+            $("#searchForm input").should(SelenideAddons.matchAttribute("placeholder", "\\d+"));
         });
     }
 
     @Test
     public void testMatchAttributeConditionErrorMissingAttribute()
     {
-        openBlogPage();
-        $("#masthead .search-toggle").click();
+        openPostersStartPage();
+        $("#header-search-trigger").click();
 
         Assert.assertThrows(ElementShould.class, () -> {
-            $("#search-container .search-field").should(SelenideAddons.matchAttribute("foo", "bar"));
+            $("#searchForm input").should(SelenideAddons.matchAttribute("foo", "bar"));
         });
     }
 
     @Test
     public void testMatchesValueCondition()
     {
-        openBlogPage();
-        $("#masthead .search-toggle").click();
-        $("#search-container .search-field").val("searchphrase").submit();
+        openPostersStartPage();
+        $("#header-search-trigger").click();
+        $("#searchForm input").val("bear").submit();
 
-        $("#content .search-field").should(SelenideAddons.matchesValue("earchphras"));
+        // used "#searchForm input" because it has the search term as value
+        $("#searchForm input").should(SelenideAddons.matchesValue("ea"));
     }
 
     @Test
     public void testMatchValueCondition()
     {
-        openBlogPage();
-        $("#masthead .search-toggle").click();
-        $("#search-container .search-field").val("searchphrase").submit();
+        openPostersStartPage();
+        $("#header-search-trigger").click();
+        $("#searchForm input").val("bear").submit();
 
-        $("#content .search-field").should(SelenideAddons.matchValue("^s.a.c.p.r.s.$"));
-        $("#content .search-field").should(SelenideAddons.matchValue("\\D+"));
+        $("#searchForm input").should(SelenideAddons.matchValue("b.a."));
+        $("#searchForm input").should(SelenideAddons.matchValue("\\D+"));
     }
 
     @Test
     public void testMatchValueConditionError()
     {
-        openBlogPage();
-        $("#masthead .search-toggle").click();
-        $("#search-container .search-field").val("searchphrase").submit();
+        openPostersStartPage();
+        $("#header-search-trigger").click();
+        $("#searchForm input").val("bear").submit();
 
         Assert.assertThrows(ElementShould.class, () -> {
-            $("#content .search-field").should(SelenideAddons.matchValue("\\d+"));
+            $("#searchForm input").should(SelenideAddons.matchValue("\\d+"));
```

Comment:
this validation should check the content area e.g. '.header-container #titleSearchText.